tensorflow/core/common_runtime/BUILD

```
@@ -1251,6 +1251,24 @@ cc_library(
     alwayslink = 1,
 )
 
+cc_library(
+    name = "zen_layout_pass",
+    srcs = ["zen_layout_pass.cc"],
+    copts = tf_copts() + ["-fopenmp"],
```

Vanilla TF shouldn't use OpenMP. I don't think `zen_layout_pass` needs OpenMP so we can just remove it.
```suggestion
    copts = tf_copts(),
```